frontend/src/app/office-hours/office-hours.models.ts

```diff
@@ -0,0 +1,214 @@
+/**
+ * @author Madelyn Andrews, Sadie Amato, Bailey DeSouza, Meghan Sun
+ * @copyright 2024
+ * @license MIT
+ */
+
+import { Section, SectionMember } from '../academics/academics.models';
+import { Room } from '../coworking/coworking.models';
+import { UserSummary } from '../models.module';
+
+export enum TicketState {
```

Comment:
Might be good to document some of these where needed!

Reply:
I documented all of the enums, do you think the other models should be documented as well?